src/pages/GamePage/GamePage.tsx

```diff
@@ -84,26 +87,38 @@ export const GamePage: FC = () => {
                 }
             });
         }
+
+        return () => {};
     }, [score]);
 
     return (
         <Styled.Wrapper>
             <GameField
-                fieldHeight={fieldHeight}
-                fieldWidth={fieldWidth}
-                endTime={endTime}
+                reachedKeys={gameProps.reachedKeys}
                 setScore={setScore}
+                lvlNumber={gameProps.lvlNum}
+                initPoint={gameProps.initPoint}
             />
+
             <Styled.GamePanel>
                 <BackButton size="s">
                     <Link to="/">На главную страницу</Link>
                 </BackButton>
                 <BaseButton onClick={() => backgroundMusic.toggleMusic()}>
-                    Toggle music
+                    Вкл/выкл музыку
+                </BaseButton>
+                <BaseButton onClick={() => setHelperState(!helperOpened)}>
+                    {helperOpened ? 'Закрыть' : 'Открыть'} подсказки
+                </BaseButton>
+                <BaseButton onClick={() => toggleFullscreen()}>
```

Comment:
❓
`toggleFullscreen` не получает параметров, зачем его заворачивать внутрь анонимной стрелки? Кажется, что тут надо сделать так
```suggestion
                <BaseButton onClick={toggleFullscreen}>
```

А сам `toggleFullscreen` обернуть в `useCallback`